src/common/ZulipSwitch.js

```diff
@@ -49,7 +49,7 @@ export default class ZulipSwitch extends PureComponent<Props, State> {
       <Switch
         value={switchValue}
         onTintColor={BRAND_COLOR}
-        tintColor={BRAND_COLOR}
+        tintColor={BRAND_OFF_COLOR}
```

Comment:
I don't think we need this as a constant. 
You can just use the value directly.